changelogs/fragments/nios_host_record-fix-aliases-removal.yml

```
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - nios_host_record - fix to remove aliases (https://github.com/ansible-collections/community.general/issues/1335).
```

```suggestion
  - nios_host_record - fix to remove ``aliases`` for configuration comparison (https://github.com/ansible-collections/community.general/issues/1335).
```
To distinguish the option ``aliases`` from the word "aliases", which could also make sense here :)